README.md

```diff
@@ -20,9 +20,390 @@ Or install it yourself as:
 
     $ gem install cassia-ruby
 
-## Usage
+## Configuration
+Here is a list of the available configuration options and their default values
 
-TODO: Write usage instructions here
+| Option          | Description                                                                                                                                                                                                                                                | Default Value                                |
+|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|
+| `ac_url`        | The URL of your AC server address                                                                                                                                                                                                                                  | `"http://iteratelabs.cassia.pro"` |
+| `client_id`     | Your Cassia API Developer Key                                                                                                                                                                                                                                  | `iteratelabs`                                          |
+| `secret`       | Your Cassia API Developer Secret                                                                                                                                                           |
+ `7d8b99e989344b6f` |
+
+### Setting your configuration
+
+To set configuration options use the `Cassia.configure` method:
+
+```ruby
+Cassia.configure do |config|
+  config.ac_url = ENV['CASSIA_AC_URL']
+  config.client_id = ENV['CASSIA_CLIENT_ID']
+  config.secret = ENV['CASSIA_SECRET']
+end
+```
+
+## Resources
+
+### AccessControllers
+
+#### Retrieve An Access Token From the Cassia API
+```ruby
+token = access_controller.get_token
+```
+`access_controller` is a `Cassia::AccessController` object which contains attributes and structures of a Cassia access controller. 
+
+`get_token` will return the `access_token` attribute of `access_controller` if it is not `nil`. Otherwise, it will generate an access token encoded from `config.client_id` and `config.secret` using base64. 
```

Comment:
This is not necessarily true, it will not return the `access_token` it returns only `true` or `false`. You have to call `access_controller.access_token` to get the value. 

`get_token` will make a request to get a new access token from the access controller using the client_id and secret